lib/players/human.rb

```diff
@@ -0,0 +1,9 @@
+class Human < Player
+
+  def move(board)
+    puts "Please choose a move by choosing a number."
+    input = gets.strip
+    input
+  end
```

Comment:
Because of Ruby's implicit return, `input` is unnecessary in this method. It could be simplified to
```
def move(board)
  puts "Please choose a move by choosing a number."
  input = gets.string
end
```
But since `input` isn't being used elsewhere in this method, it could be further simplified to 
```
def move(board)
  puts "Please choose a move by choosing a number."
  gets.strip
end
```